syntax = "proto3";
message GasThreshold{
	
    string  Name  = 1;        
	float  ThresholdL_Max1 = 2;  
 	float  ThresholdL_Max2 = 3;
 	float  ThresholdL_Max3 = 4;      
	float  ThresholdL_Min1 = 5;  
 	float  ThresholdL_Min2 = 6;
	float  ThresholdL_Min3 = 7;       
    float  ThresholdChange = 8;
}
message GasThresholdList {
  repeated GasThreshold thresholds = 1;
}